README.md

```diff
@@ -572,14 +567,168 @@ Here's how you can configure it:
 
 This configuration option may not be changed after installation.
 
+### Recovery and Health Checks
+
+You can enable automated replacement of brokers and configure the circumstances under which they are replaced.
+
+#### Enable Broker Replacement
+
+To enable automated replacement: <!-- what's all this about data loss, then? -->
+
+* **DC/OS CLI options.json**:
+
+```json
+    {
+        "enable_replacement":{
+            "description":"Enable automated replacement of Brokers. WARNING: May cause data loss. See documentation.",
```

Comment:
Is there more to say about this data loss? Is there a way to prevent it? Is there a particular area of the documentation a user should consult? I don't think we have any information on backing up data.